documentation/docs/examples/cad-nodefiltering.mdx

```diff
@@ -0,0 +1,61 @@
+---
+id: cad-nodefiltering
+title: Filtering CAD nodes
+describe: Change the appearance and visibility of nodes based on node properties
+---
+
+import { LiveCodeSnippet } from '../../src/components/LiveCodeSnippet';
+import { DemoWrapper } from '../../src/components/DemoWrapper';
+
+
+When CAD models are ingested in [Cognite Data Fusion (CDF)](https://docs.cognite.com/cdf/), all node properties
+(also called attributes) are stored and indexed for efficient filtering. This allows applications to perform 
+queries and change the appearance of the 3D model based on the result.
+
+<DemoWrapper
+  modelId={4715379429968321}
+  revisionId={5688854005909501}
+  name="Cognite3DViewerDemo"
+/>
+
+## Only show nodes that has a given node attribute value
+
+The available node properties varies a lot from model to model, and to use this information you will need to investigate
+the structure of the underlying models. For the purpose of this example, we the attributes of the nodes typically looks something like:
+```json
+"PDMS": {
+  ":capDisc": "GENE",
+  "Function": "EA",
+  "Type": "SITE",
+  "Description": "EARLY WORK",
+  ":capPdmsProjectId": "VPH-RIGE",
+  "Position": "E 0mm N 0mm U 0mm",
+  ":capDbName": "MMOSTD03",
+  "Purpose": "EA",
+  "Module": "A00",
+  "Date": "5  Jun 2018 17:42"
+}
+```
+
+To query for nodes matching a given attribute value, we use [list3DNodes](https://cognitedata.github.io/cognite-sdk-js/classes/revisions3dapi.html#list3dnodes)
+from the [Cognite SDK](https://www.npmjs.com/package/@cognite/sdk) which maps to the [List 3D Nodes API endpoint](https://docs.cognite.com/api/v1/#operation/get3DNodes).
+For this example we query for nodes that are within the 'EP' function and hide all other nodes.
+
+<LiveCodeSnippet>
+{`
+model.hideAllNodes();
+sdk.revisions3D.list3DNodes(model.modelId, model.revisionId, 
+  { 
+    limit: 1000, 
+    properties: { PDMS: { Function: 'EA' } }
```

Comment:
I suggest using something more readable to demonstrate the point that we can filter properties. Function "EA" doesn't make any sense to me personally, so it's just harder to read than some filtering by name/type/date or whatever more meaningful.

Reply:
I agree that this doesn't make much sense, but I haven't really been able to find good examples of filters I think make sense. The input model is not great when it comes to node metadata so finding good examples is hard.